core-java-modules/core-java-jndi/src/main/java/com/baeldung/jndi/ldap/connectionTool/LdapConnectionTool.java

```diff
@@ -0,0 +1,82 @@
+package com.baeldung.jndi.ldap.connectionTool;
```

Comment:
If I´m not wrong the package convention says that the package name should be lowercase.

Reply:
Nice catch! PR updated with correct package naming.